### ISSUE TITLE
Zend Framework

### ISSUE THREAD

**Issue by kapilsharma** [komarserjio/notejam#44]
READMe page mention Zend (Framework) is in progress but it do not mention if it is ZF1 or ZF2.

If it is ZF2, I'd like to suggest (or even contribute) ZF1.

It might seems ZF2 is more preferred but ZF is used for enterprise applications. Although ZF2 is preferred for any new project, ZF1 projects could not be migrated to ZF2 without major rewriting, therefore are lot of ZF1 enterprise projects and I'm still working on one. If current ZF project is ZF2 project, I'd like to contribute to ZF1 branch.

**Comment by komarserjio** [comment]
I would appreciate any help :) 
ZF1 is still actively used, it makes sense to have it on board. 

@kapilsharma Ping me if any questions. I have ZF1 experience in the past, so I can help :).

**Comment by kapilsharma** [comment]
Great @komarserjio 
I'm revising my ZF1 basics so picked up notejam to do so. this morning, I already started it on fork (branch & folder: zendframework1).
Let me complete minimal things and I'll raise pull request so that you can also contribute. Or if needed, I can do it on main repo as well.

**Comment by komarserjio** [comment]
Looking forward for the PR :)
P.S Do not forget to prepend your commits with a prefix (I guess `ZF1`)

**Comment by komarserjio** [comment]
Also we need to add wrapper to index.php to make in runnable by php built-in web server. 
Official docs offers Zend Server for quick launch, but it's absolutely unacceptable. I'm pretty sure it's possible to make it runnable by php's built-in web server

**Comment by komarserjio** [comment]
This looks like a solution http://arr.gr/blog/2012/08/serving-zf-apps-with-the-php-54-built-in-web-server/

**Comment by kapilsharma** [comment]
Yes @komarserjio I'll try that once basics functionality is done. Got little busy. We have had code night in office yesterday and while working on an issue we discovered a bug in ZF1 view helper. Applied patch on official project but this weekend working on its patch for ZF, and if accepted, will be my first contribution to ZF1.

I'm planning to do it is as follow:

- ZF1 view helper patch - This weekend (18-19 July)
- Notejam functionality - Next 2 weekend (deadline 2 August)
- Notejam - Fixing code review comments, best practices and extras - (deadline 9 August)

Keeping one weekend as buffer for unexpected/unplanned personal/family things, I am expecting to finish it by 16th August, including PHP in-build server. Does it seems good to you?

**Comment by komarserjio** [comment]
Perfect, looks very ambitious!
It's worth to mention there is no hurry :)

Good luck with your first contribution to ZF :smile:

**Comment by komarserjio** [comment]
Hey @kapilsharma. How is it going? :) Do you need any help?

**Comment by kapilsharma** [comment]
Hi @komarserjio. Sorry for late reply. Last weekend, i created template, finished starting docs, phpunit and db migration. This weekend, I'll finish auth and most pages.

You can check latest status on my fork. Except few files, most things pushed to fork. If needed, I can send pull request, and work further from this repo. If you are free, you can help with docs and php units (working or incomplete)

**Comment by komarserjio** [comment]
Create pull request when you have more or less working prototype and we'll continue work together in the main repo.
